src/Symfony/Component/Security/Core/Security.php

```diff
@@ -65,12 +66,22 @@ public function getUser(): ?UserInterface
     }
 
     /**
-     * Checks if the attributes are granted against the current authentication token and optionally supplied subject.
+     * Checks if the attribute is granted against the current authentication token and optionally supplied subject.
      */
-    public function isGranted(mixed $attributes, mixed $subject = null): bool
+    public function isGranted(mixed $attribute, mixed $subject = null): bool
     {
         return $this->container->get('security.authorization_checker')
-            ->isGranted($attributes, $subject);
+            ->isGranted($attribute, $subject);
+    }
+
+    /**
+     * Checks if the attribute is granted against the user and optionally supplied subject.
+     * This should be used over isGranted() when checking permissions against a user that is not currently logged in.
+     */
+    public function userIsGranted(UserInterface $user, mixed $attribute, mixed $subject = null): bool
+    {
+        return $this->container->get('security.user_authorization_checker')
+            ->userIsGranted($user, $attribute, $subject);
     }
```

Comment:
The `Security` class in the SecurityCore component is supposed to die. It only exists for BC. There's a new `Security` class in SecurityBundle which is meant to be the replacement. I think, if we introduce a new method, it should go into the new class.